test/rekt/eventtype_test.go

```diff
@@ -0,0 +1,49 @@
+//go:build e2e
+// +build e2e
+
+/*
+Copyright 2020 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package rekt
+
+import (
+	"testing"
+	"time"
+
+	"eventing/test/rekt/features/eventtype"
```

Comment:
Question: I added the new module `eventtype` in rekt. How should I import the module here?

Please lmk if this is not the right way to approach this ticket. 

/cc @matzew @pierDipi @creydr

Reply:
Seems you're using an incorrect import path. The knative modules are all under `knative.dev/<repo>/...` (no matter if it is a knative-sandbox project or not). So in this case it would be `"knative.dev/eventing/test/rekt/features/eventtype"`

Reply:
Regarding the approach: In the test files (under test/rekt/*/*_test.go), we usually setup only the environment and call the feature test (as you have done). In the feature test (`test/rekt/features/`) we have the according test implementation.

Reply:
I tried that before. If I use `knative.dev/eventing/test/rekt/features/eventtype`, it will throw this error

`test/rekt/eventtype_test.go:47:32: undefined: eventtype.eventTypeWithBrokerAsReference`

Does importing the packages that starting with `knative.dev/***` will actually download the latest knative package from the source? Because this `eventTypeWithBrokerAsReference` is a new function that I added in this PR, it hasn't been merged into the main branch. @creydr

Reply:
It's probably, because you have to make `eventTypeWithBrokerAsReference` public